OneginiExampleAppSwift/ViewControllers/Login/LoginViewController.swift

```diff
@@ -14,30 +14,30 @@
 // limitations under the License.
 
 import UIKit
+import OneginiSDKiOS
 
 protocol LoginViewDelegate: class {
     
-    func loginView(_ loginView: UIViewController, didLoginProfile profile: ONGUserProfile, withAuthenticator authenticator: ONGAuthenticator?)
-    func loginView(profilesInLoginView loginView: UIViewController) -> [ONGUserProfile]
-    func loginView(_ loginView: UIViewController, authenticatorsForProfile profile: ONGUserProfile) -> [ONGAuthenticator]
-    func loginView(_ loginView: UIViewController, implicitDataForProfile profile: ONGUserProfile, completion: @escaping (String?) -> Void)
+    func loginView(_ loginView: UIViewController, didLoginProfile profile: UserProfile, withAuthenticator authenticator: Authenticator?)
+    func loginView(profilesInLoginView loginView: UIViewController) -> [UserProfile]
```

Comment:
`func profilesInLoginView(_ loginView: UIViewController) -> [UserProfile]`

Reply:
This was fixed.

Reply:
don't you see the difference between how you changed it and the way it should be changed? :)

Reply:
A, ok, too many letters :)